lib/game.rb

```diff
@@ -0,0 +1,119 @@
+require_relative "./player.rb"
+require_relative "./players/human.rb"
+
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+
+  end
+
+  def board
+    @board
+  end
+
+  def player_1
+    @player_1
+  end
+
+  def player_2
+    @player_2
+  end
+
+  def turn_count
+    turn=0
+    board.cells.each do |index|
+      if index=="X"|| index=="O"
+        turn+=1
+      end
+    end
+    turn
+  end
+
+  def current_player
```

Comment:
You can use a single line ternary inside this method. 

`@board.turn_count.even? ? @player_1 : @player_2`